frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -67,20 +68,36 @@ export const funnelLogic = kea({
                 if (props.cachedResults && !refresh && values.filters === props.filters) {
                     return props.cachedResults
                 }
+
                 const { from_dashboard } = values.filters
                 const cleanedParams = cleanFunnelParams(values.filters)
                 const params = {
                     ...(refresh ? { refresh: true } : {}),
                     ...(from_dashboard ? { from_dashboard } : {}),
                     ...cleanedParams,
                 }
+
                 let result
 
                 insightLogic.actions.startQuery()
+
+                const eventName = 'calculate funnel result'
+                const eventProps = {
+                    'event count': params.events.length,
+                    'action count': params.actions.length,
+                    'total count actions and events': params.actions.length + params.events.length,
+                    interval: params.interval,
+                }
+
                 try {
                     result = await pollFunnel(params)
+                    eventProps.success = true
+                    posthog.capture(eventName, eventProps)
                 } catch (e) {
                     insightLogic.actions.endQuery(ViewType.FUNNELS, false, e)
+                    eventProps.success = false
+                    eventProps.error = e
```

Comment:
Think we might need to convert the error to string?

Reply:
going to use `e.name` instead here